nexus/tests/output/nexus_tags.txt

```diff
@@ -76,115 +78,98 @@ timeseries_schema_get                    /timeseries/schema
-OPERATION ID                             URL PATH
-routers_routes_delete_route              /organizations/{organization_name}/projects/{project_name}/vpcs/{vpc_name}/routers/{router_name}/routes/{route_name}
-routers_routes_get                       /organizations/{organization_name}/projects/{project_name}/vpcs/{vpc_name}/routers/{router_name}/routes
-routers_routes_get_route                 /organizations/{organization_name}/projects/{project_name}/vpcs/{vpc_name}/routers/{router_name}/routes/{route_name}
-routers_routes_post                      /organizations/{organization_name}/projects/{project_name}/vpcs/{vpc_name}/routers/{router_name}/routes
-routers_routes_put_route                 /organizations/{organization_name}/projects/{project_name}/vpcs/{vpc_name}/routers/{router_name}/routes/{route_name}
+role_list                                /roles
+role_view                                /roles/{role_name}
 
 API operations found with tag "sagas"
 OPERATION ID                             URL PATH
-sagas_get                                /sagas
-sagas_get_saga                           /sagas/{saga_id}
+saga_list                                /sagas
+saga_view                                /sagas/{saga_id}
 
-API operations found with tag "silos"
+API operations found with tag "session"
 OPERATION ID                             URL PATH
-silo_saml_idp_create                     /silos/{silo_name}/saml_identity_providers
-silo_saml_idp_fetch                      /silos/{silo_name}/saml_identity_providers/{provider_name}
-silo_users_get                           /users
-silos_delete_silo                        /silos/{silo_name}
-silos_get                                /silos
-silos_get_identity_providers             /silos/{silo_name}/identity_providers
-silos_get_silo                           /silos/{silo_name}
-silos_get_silo_policy                    /silos/{silo_name}/policy
-silos_post                               /silos
-silos_put_silo_policy                    /silos/{silo_name}/policy
+session_sshkey_create                    /session/me/sshkeys
+session_sshkey_delete                    /session/me/sshkeys/{ssh_key_name}
+session_sshkey_list                      /session/me/sshkeys
+session_sshkey_view                      /session/me/sshkeys/{ssh_key_name}
 
-API operations found with tag "sleds"
+API operations found with tag "silos"
 OPERATION ID                             URL PATH
-hardware_sleds_get                       /hardware/sleds
-hardware_sleds_get_sled                  /hardware/sleds/{sled_id}
+silo_create                              /silos
+silo_identity_provider_create            /silos/{silo_name}/saml_identity_providers
+silo_identity_provider_list              /silos/{silo_name}/identity_providers
+silo_identity_provider_view              /silos/{silo_name}/saml_identity_providers/{provider_name}
+silo_list                                /silos
+silo_list_users                          /users
```

Comment:
wonder if we should rename `/users` to `/silo_users` so it's immediately apparent that it's not a global list

Reply:
For the record, there's a bunch of discussion about this in chat right now and I think we probably want to open it for other folks that might have thoughts.  I think there's a broader question of how much end users are expected to know about Silos and also what they're allowed to know.

Reply:
Yeah, I both agree and disagree with the rename. In the context of the existing patterns here it's quite obvious that the rename would make sense. Or rather that it would be shaped like `/silos/{silo_name}/users`. Contextually though having a `/users` route that _also_ exists (which an operation ID that matches) would be beneficial from an end user perspective. Hopefully after the operator API discussion today we can hone down on how to shape this.

Reply:
My current understanding of this is that every user exists in a silo. Given that, a lot of top level calls happen in the context of the current silo. `/organizations/`, `/session/me`, etc. Some things do not like `/hardware`, and `/images`. 

I think we're lacking clarity here that could be solved in a couple of ways:
1. (my preference) would be to have a set of top routes that are understood to be outside the context of a silo. `fleet` and `silos` are the most obvious to me. `/fleet/hardware` and  `/fleet/images` are some examples which I think help clarify the scope of certain resources. Otherwise it would be assumed that top level routes are in the context of the silo in which the current user is authenticated for. 
2. We make everything very explicit. e.g. `/silos/{silo}/organizations/{org}/projects/{project}`. In that way we'd assume that everything is defaulted explicitly to the fleet level. While this may be the clearest in that global really does mean global and what you see is what you get from the URL perspective it also makes for an odious experience when you're just trying to do things with a resource on the rack. I know other providers don't have the best of URLs, but we do have an opportunity here to be better than the example they set.

I'm sure there are other solutions that haven't occurred to me. It does seem though that right now we're lacking an overall clarity of at what level a resource actually occupies which seems like an undesirable state.

Reply:
Of course, it goes without saying that this is a larger change and not in the scope of this PR, ha.

Reply:
Notably [RFD 234](https://rfd.shared.oxide.computer/rfd/0234#_silos) says this:

> Identities for users, groups, and service accounts are always scoped to a particular Silo. As a result, there’s no need for Silos to appear in the API URL for Organizations, Projects, etc. This is an important isolation measure: there’s no way for users to even ask about resources in other Silos — they don’t have a way to refer to them.

It seems we're already sort of targeting point 1, we may just need to make it more formalized.

Reply:
From this discussion, I'm going to set the `operationId` for `/users` to simply `user_list` (it will imply the current silo for the active user).

To add to the list of outside-of-silos things: silos themselves, hardware, global images, **also** some external networking config (not sure where we configure this), and updates.